### DISCUSSION TITLE
How to write this specific propagator in Mathematica/FeynCalc (in image file)?

### DISCUSSION THREAD

**Discussion by alessiotbceleste** [FeynCalc/feyncalc#270] [General]
![propagator](https://github.com/FeynCalc/feyncalc/assets/136126565/1f6a6efc-1b63-41f4-95ce-b3284aa26e98)

**Comment by vsht** [comment]
In this form you can't. I suppose that the propagator can be massaged into something more conventional without containing implicit inverse matrices.

Am 23. Mai 2024 02:37:56 UTC schrieb alessiotbceleste ***@***.***>:
